frame/staking/src/benchmarking.rs

```diff
@@ -243,7 +243,7 @@ benchmarks! {
 
 		// all nominators now should be nominating our validator...
 		for n in nominator_stashes.iter() {
-			assert!(Nominators::<T>::get(n).unwrap().targets.contains(&stash));
+			assert!(Nominators::<T>::get(n).unwrap().targets.iter().map(|(t, _)| t).collect::<Vec<_>>().contains(&&stash));
```

Comment:
iterator [any](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.any) should allow you to avoid that allocation.